lua/cfc_entity_stubber/m9k/assault_rifles/m9k_acr.lua

```diff
@@ -0,0 +1,14 @@
+if SERVER then AddCSLuaFile() end
+
+CFC_M9k_Stubber.registerStub( function()
```

Comment:
All these files are using `CFC_M9k_Stubber.registerStub` this wont work.
Use `cfcEntityStubber.registerStub` like shown in the examples.